src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1050,6 +1057,7 @@ namespace UnitarySystems {
                 } else {
                     this->m_DesignSuppHeatingCapacity = 0.0;
                 }
+                state.dataLoopNodes->Node(this->m_SuppCoilAirInletNode).MassFlowRate = saveCoilAirflow;
```

Comment:
I noticed that the air flow used, when testing what the supplemental heating coil capacity is for this time step, is 0. This in turn sets this->m_DesignSuppHeatingCapacity = 0 for this time step and the supp coil cannot turn on. I also added the save air flow during this time and reset it at the end which I don't think is necessary since when the system is simulated the conditions will pass from UnitarySystem inlet to outlet and this value will be overwritten. So I think I can safely remove lines 1038 and 1060.